### ISSUE TITLE
How to replace the font?

### ISSUE THREAD

**Issue by rahaaatul** [Teraskull/bigsur-grub2-theme#8]
Title says it all. I'm a windows user and I'm unfamiliar with `pf2` fonts. I searched a bit and found that it needs to be converted from `ttf` to `pf2`, but the tools are in Linux. 

But anyway, will replacing the font change the fonts? Are there any specific guidelines to follow for your theme?

**Comment by Teraskull** [comment]
@rahaaatul By menu-tip you mean the ISO entries? I don't know if 's possible.

**Comment by rahaaatul** [comment]
> @rahaaatul By menu-tip you mean the ISO entries? I don't know if 's possible.

Yes.

```JSON
    "menu_tip":{
        "left": "5%",
        "top": "95%",
        "color": "#eff0f1",
        "tips":[
            {
                "image": "/images/linux/pop-os_22.04_amd64_intel_32.iso",
                "tip": "Version: 22.04 LTS"
            }
        ]
    },
```

**Comment by Teraskull** [comment]
@rahaaatul oh, I'm unfamiliar with the Menu Tip Ventoy plugin. I don't think you can center it automatically.

**Comment by nicholasz2510** [comment]
https://github.com/AdisonCavani/distro-grub-themes/issues/101#issuecomment-1423353475

tl;dr having secure boot enabled makes it so external fonts don't work :/
